src/engine/source/cmds/include/cmds/cmdRun.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef _CMD_RUN_HPP
+#define _CMD_RUN_HPP
+
+#include <string>
+
+namespace cmd
+{
+
+/**
+ * @brief Run environment.
+ *
+ * @param kvdbPath Path to KVDB folder.
+ * @param endpoint Endpoint of the server.
+ * @param queueSize Size of the event ingestion queue.
+ * @param threads Number of environment threads.
+ * @param fileStorage Path to asset folders.
+ * @param environment Name of the environment to be loaded.
+ */
+void run(const std::string& kvdbPath,
+         const std::string& endpoint,
+         const int queueSize,
+         const int threads,
+         const std::string& fileStorage,
+         const std::string& environment,
+         const int logLevel);
+} // namespace cmd
```

Comment:
Using a structure to group configurations.

Reply:
why?